src/videojs-fairplay.js

```diff
@@ -128,11 +129,42 @@ class Html5Fairplay {
   onCertificateLoad(event, { callback }) {
     this.log('onCertificateLoad()');
 
-    certificate = new Uint8Array(event.target.response);
+    const {
+      response,
+      status,
+    } = event.target;
+
```

Comment:
wondering if this is an issue, not sure if response will always be binary, can fix if error later.